pihole

```diff
@@ -535,7 +541,6 @@ Switch Pi-hole subsystems to a different Github branch
 Repositories:
   core [branch]       Change the branch of Pi-hole's core subsystem
   web [branch]        Change the branch of Admin Console subsystem
-  ftl [branch]        Change the branch of Pi-hole's FTL subsystem
```

Comment:
Oh shoot, I edited my local `pihole` and not from the latest Development. I have no idea how I missed that :sweat_smile: